homeassistant/components/deconz/climate.py

```diff
@@ -33,20 +33,20 @@
 DECONZ_FAN_SMART = "smart"
 
 FAN_MODES = {
-    DECONZ_FAN_SMART: "smart",
-    FAN_AUTO: "auto",
-    FAN_HIGH: "high",
-    FAN_MEDIUM: "medium",
-    FAN_LOW: "low",
-    FAN_ON: "on",
-    FAN_OFF: "off",
+    "smart": DECONZ_FAN_SMART,
+    "auto": FAN_AUTO,
+    "high": FAN_HIGH,
+    "medium": FAN_MEDIUM,
+    "low": FAN_LOW,
+    "on": FAN_ON,
+    "off": FAN_OFF,
```

Comment:
That is not the inversion Martin was referring to. e.g.

```
DECONZ_TO_FAN_MODES = {
    DECONZ_FAN_SMART: "smart",
    FAN_AUTO: "auto",
    FAN_HIGH: "high",
    FAN_MEDIUM: "medium",
    FAN_LOW: "low",
    FAN_ON: "on",
    FAN_OFF: "off",
}

FAN_MODE_TO_DECONS = {value: key for key, value in DECONZ_TO_FAN_MODES.items()}
```

And next use the appropriate constant below to do an instant lookup.

Reply:
Alright

Reply:
Fixed